### ISSUE TITLE
Isn't this filter a duplicate effort to `pandoc-csv2table` ?

### ISSUE THREAD

**Issue by pdfkungfoo** [mb21/pandoc-placetable#1]
You seem not to be aware of this filter:
- [pandoc-csv2table](https://github.com/baig/pandoc-csv2table) (born 2 months ago)

This does a very similar thing to yours (born 2 days ago).

Wouldn't it be better if you two guys (@mb21 ad @baig) pooled your efforts into one?

**Comment by mb21** [comment]
Yes, I actually tried to use it but hit [a bug](https://github.com/baig/pandoc-csv2table/issues/13). When I looked into fixing the bug, the decision to generate the table as a markdown string instead of directly generating a Pandoc AST struck me as quite error-prone and laborious, but it is a quite fundamental design decision in the filter, so I decided to roll my own, which:
- is only ~100 lines of code instead of ~450
- uses the `spreadsheet` package instead of `csv`, thus supports specifying custom delimiters
- has a compile flag to not require `pandoc` as a dependency, so building/cabal-installing without support for inline markdown is much quicker if you installed pandoc as a binary. but when compiled with inlinemarkdown support, there is also the `inlinemarkdown=yes` option to turn this on/off per csv.
